src/select.rs

```diff
@@ -87,3 +92,40 @@ impl<'a> TElement<'a> for &'a ElementData {
         }
     }
 }
+
+impl<'a> Node<'a> {
+    pub fn css(&'a self, css_str: &str) -> FilterNodes<'a, Descendants<'a>> {
```

Comment:
I was just copying what I thought Nokogiri did, with their `css` method. Originally I thought to go with `select`, but what if XPath is ever added to Kuchiki, `select` would IMHO be confusing (select by XPath or using CSS selectors)?